archr/targets/docker_target.py

```diff
@@ -104,6 +105,9 @@ def build(self, pull=False):# pylint:disable=arguments-differ
             (self.image.attrs['Config']['Entrypoint'] or [ ]) + (self.image.attrs['Config']['Cmd'] or [ ])
         )
 
+        if not self.target_args:
+            raise ArchrError("Canot build: target_args is empty!")
```

Comment:
This I feel like should have a more specific exception as well, maybe inheriting from ValueError and ArchrError?

Reply:
So `target_args` is either set by the user at `init` or it is taken from the image default.  My use case involves running default args image args, which may be None.  I don't think the user would set it to none..

Any suggestion for this exception's name?  `ArchrValueError`?  AFAIK it's an edge-case that only exists here.